hibernate-core/src/test/java/org/hibernate/orm/test/mapping/converted/converter/EntityWithEnum.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
+ */
+package org.hibernate.orm.test.mapping.converted.converter;
+
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+
+/**
+ * @author Jan Schatteman
+ */
+@Entity(name = "EntityWithEnum")
+public class EntityWithEnum {
```

Comment:
It's better to refactor it as inner class of `AttributeConverterAndFunctionTest`, since there are other inner domain classes.

Reply:
That would be true, but the hibernate processes doesn't handle inner classes